src/styles/stars.css

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
```

Comment:
Remove the space between `.stars--3` and `.stars__star:nth-child(-n + 3)` to ensure you're targeting direct children only.